Wikipedia/Code/ExploreViewController.swift

```diff
@@ -411,12 +411,7 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         cell.titleLabel.text = group.headerTitle
         cell.subtitleLabel.text = group.headerSubTitle
         cell.footerButton.setTitle(group.moreTitle, for: .normal)
-        switch group.contentGroupKind {
-        case .relatedPages, .locationPlaceholder:
-            cell.customizationButton.isHidden = false
-        default:
-            cell.customizationButton.isHidden = true
-        }
+        cell.customizationButton.isHidden = !group.contentGroupKind.isCustomizable
```

Comment:
The customization button is missing on "Because you read" cards. Maybe because that kind `isGlobal` but not `isCustomizable`